app/views/events/_game_selection.html.haml

```diff
@@ -1 +1 @@
-= collection_select(:event, :game_id, Game.all, :id, :name, {:include_blank => 'Kein Game ausgewählt ...'}, { class: 'custom-select custom-select-sm' })
\ No newline at end of file
+= collection_select(:event, :game_id, [], :id, :name, {:include_blank => 'Kein Game ausgewählt ...'}, { class: 'custom-select custom-select-sm', id: 'game-search' })
```

Comment:
Change param value to igdb_game_id or igdb_id. New model attribute will be created in distinct branch